docs-v2/integrate/guides/receive-webhooks-from-an-api.mdx

```diff
@@ -53,16 +53,16 @@ Each modified record (following an external webhook) will trigger a separate web
     "providerConfigKey": "<INTEGRATION-ID>",
     "syncName": "<SYNC-NAME>",
     "model": "<MODEL-NAME>",
-    "responseResults": { "<MODEL-NAME>": { "added": 1, "updated": 0, "deleted": 0 } },
+    "responseResults": { "added": 1, "updated": 0, "deleted": 0 },
```

Comment:
Confirmed that a multi model sync sends multiple webhooks:
```
[2] 2024-04-11T07:31:29.176Z [INFO] [Activity] Sync webhook sent successfully and received with a 200 response code to https://yahoo.com with the following data: {
[2]   "from": "nango",
[2]   "type": "sync",
[2]   "connectionId": "un",
[2]   "providerConfigKey": "unauthenticated",
[2]   "syncName": "fluff-sync",
[2]   "model": "Test",
[2]   "responseResults": {
[2]     "added": 1,
[2]     "updated": 0,
[2]     "deleted": 0
[2]   },
[2]   "syncType": "INCREMENTAL",
[2]   "modifiedAfter": "2024-04-11T07:31:25.840Z",
[2]   "queryTimeStamp": "2024-04-11T07:31:25.840Z"
[2] }
[2] 2024-04-11T07:31:29.178Z [INFO] [Activity] The INCREMENTAL "fluff-sync" sync has been completed to the Test model. The version integration script version ran was 2. The result was 1 added record.
[2] 2024-04-11T07:31:30.643Z [INFO] [Activity] Sync webhook sent successfully and received with a 200 response code to https://yahoo.com with the following data: {
[2]   "from": "nango",
[2]   "type": "sync",
[2]   "connectionId": "un",
[2]   "providerConfigKey": "unauthenticated",
[2]   "syncName": "fluff-sync",
[2]   "model": "GithubIssue",
[2]   "responseResults": {
[2]     "added": 1,
[2]     "updated": 0,
[2]     "deleted": 0
[2]   },
[2]   "syncType": "INCREMENTAL",
[2]   "modifiedAfter": "2024-04-11T07:31:25.840Z",
[2]   "queryTimeStamp": "2024-04-11T07:31:25.840Z"
[2] }
[2] 2024-04-11T07:31:30.646Z [INFO] [Activity] The INCREMENTAL "fluff-sync" sync has been completed to the GithubIssue model. The version integration script version ran was 2. The result was 1 added record.
```

Reply:
Thanks 🙏🏻